build/loc/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using CommandLine;
+
+// ReSharper disable SuggestVarOrType_BuiltInTypes
+// ReSharper disable SuggestVarOrType_Elsewhere
+
+namespace OneLocBuildSetup
+{
+    [SuppressMessage("Style", "IDE0008:Use explicit type")]
+    internal class Program
+    {
+        private const string XlfExtension = ".xlf";
+        private const string SpanishXlfExtension = ".es" + XlfExtension;
```

Comment:
>SpanishXlfExtension [](start = 29, length = 19)

what was special about Spanish? a comment here would help.

Reply:
There is nothing special about Spanish. It could be any language. I just picked one at random by writing every language onto slips of paper, putting them into a hat, and then pulling one out at random... or not. 😝 The OneLocBuild system uses the structure of the file, not the content.

Reply:
Makes sense, this would be great to mention in the code though otherwise future maintainers might expect there to be something magical about ".es"

---
In reply to: [613569792](https://github.com/dotnet/project-system/pull/7108#discussion_r613569792) [](ancestors = 613569792)

Reply:
Added comments above this and link for explanation.